k8s/aat/common/rpe/pdf-service.yaml

```diff
@@ -14,12 +14,73 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: cmc-pdf-service
-    version: 0.2.1
+    version: 0.3.0
   values:
+    aadIdentityName: rpe
     java:
       replicas: 2
       useInterpodAntiAffinity: true
       applicationInsightsInstrumentKey: "cdd56761-038b-49e9-9fdf-b42b10e11ed5"
       image: hmctspublic.azurecr.io/cmc/pdf-service:prod-7ad4efac
+      smoketestscron:
+        image: hmctspublic.azurecr.io/cmc/pdf-service:test-prod-7ad4efac
+        schedule: "25 0/1 * * *"
+        environment:
+          TEST_URL: http://pdf-service-java
+          TASK: smoke
+          SLACK_CHANNEL: "rpe-build-notices"
+          SLACK_WEBHOOK: "draft-store/slack-webhook"
+          SLACK_NOTIFY_SUCCESS: "false"
+        keyVaults:
+          "draft-store":
```

Comment:
hah this one is interesting, pdf-service doesn't have a vault...

Reply:
Should we do this using sealed secrets?

Reply:
I mean using https://github.com/hmcts/chart-java#secrets and not keyvaults ?